src/readers/morphologyASC.cpp

```diff
@@ -217,6 +225,12 @@ class NeurolucidaParser
        Parse the root sexp until finding the first sexp containing numbers
     **/
     Header parse_root_sexp_header() {
+        static const std::map<std::string, Token> NeuriteStringTokenMap{
```

Comment:
for this usecase where we don t need stable iterators, don t care about ordering and having string as keys on a small map isn't an unordered_map a better choice ?

Reply:
map is gone in the latest version